s3direct/utils.py

```diff
@@ -47,10 +47,11 @@ def get_aws_v4_signature(key, message):
     return hmac.new(key, message.encode('utf-8'), hashlib.sha256).hexdigest()
 
 
-def get_key(key, file_name, dest):
+def get_key(key, user, file_name, dest):
```

Comment:
is this fixing an outdated call?

Reply:
@semanticmx I needed to pass in the user object from the request into the s3 key/filename generation function so I can place videos into a folder tied to the user's account